src/utils.h

```diff
@@ -167,4 +168,24 @@ ts_clear_flags_32(uint32 bitmap, uint32 flags)
 	return bitmap & ~flags;
 }
 
+/**
+ * Try to register a custom scan method.
+ *
+ * When registering a custom scan node, it might be called multiple times when
+ * different databases have different versions of the extension installed, so
+ * this function can be used to try to register a custom scan method but not
+ * fail if it has already been registered.
+ *
+ * @retval true The custom scan method was registered
+ * @retval false The custom scan method was not registered
+ */
+static inline bool
```

Comment:
Do we need to return `bool` since the callers doesn't do anything with the returned value?

Reply:
Was thinking that it could be useful, but let's cross that bridge when we come to it. Removed.